pkg/kubeapiserver/options/authentication.go

```diff
@@ -697,13 +716,14 @@ func (o *BuiltInAuthenticationOptions) ApplyTo(ctx context.Context, authInfo *ge
 
 				timeoutCtx, timeoutCancel := context.WithTimeout(ctx, UpdateAuthenticationConfigTimeout)
 				defer timeoutCancel()
-				if err := updateAuthenticationConfig(timeoutCtx, authConfig); err != nil {
+				if err = updateAuthenticationConfig(timeoutCtx, authConfig); err != nil {
 					klog.ErrorS(err, "failed to update authentication config")
 					// we do not update the tracker here because this error could eventually resolve as we keep retrying
 					return
 				}
 
 				trackedAuthenticationConfigData = authConfigData
```

Comment:
should we log an info message here that we reloaded authentication config successfully?

Reply:
added an info message